x-pack/plugins/observability/common/utils/formatters/duration.test.ts

```diff
@@ -5,7 +5,15 @@
  * 2.0.
  */
 
-import { asDuration, asTransactionRate, toMicroseconds, asMillisecondDuration } from './duration';
+
+import {
+  asDuration,
+  asTransactionRate,
+  toMicroseconds,
+  asMillisecondDuration,
+  formateDurationFromTimeUnitChar,
```

Comment:
@fkanout Maybe `formatDurationFromTimeUnitChar` instead ? (a small typo)

Reply:
the French language hit, again. Thanks!

Reply:
Done